app/tests/system/setup/setup.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * Loop through the custom contexts that exist when the app starts
+ * and grab their names.
+ *
+ * To change the environment we're testing against, specify it in site.json.
+ * This should align with what the app is built against, which is declared in
+ * app/environment.js hostUrl
+ */
+
+var Site = require('../site.json');
+
+var Setup = {};
+
+var selectors = {
+    previewButton: '#authorize',
+    registeredAuctionList: '.js-auction-lists'
+};
+
+var expected = {
+    //EXAMPLE Webviews constructed for project. 
+    //These expressions are match the part of the URL 
+    'noInternet': /no-internet\.html/i,
+    'couldNotLoad': /could-not-load\.html/i
+};
+
+Setup.isAndroid = function(platform) {
+    platform = platform || 'platformName';
```

Comment:
One last little nitpick.  Could we make this `'unknown'` instead of `'platformName`'?